src/redprl.mlb

```diff
@@ -73,7 +73,6 @@ in
 
 
     ann
-      "nonexhaustiveBind ignore"
```

Comment:
Which version of MLton are you using? I believe this is a new feature introduced in 2017: (see the commit about their documentation https://github.com/MLton/mlton/commit/0908f6af4db86fd88461b358411a4488560838e2). If I understand it correctly, the `nonexhaustiveMatch` was broken into multiple annotations.

Reply:
I guess my build is from 2016; I'm just using the one that you get in Ubuntu when you run `apt install mlton`.

Reply:
I see. I was using a very recent version because I want https://github.com/MLton/mlton/pull/190.

Reply:
Ah :smile: